public/404.html

```diff
@@ -3,65 +3,22 @@
 <head>
   <title>The page you were looking for doesn't exist (404)</title>
   <meta name="viewport" content="width=device-width,initial-scale=1">
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/css/bootstrap.min.css">
```

Comment:
Why are we adding bootstrap here? Isn't it already being loaded by the bootstrap-sass gem?